docs/src/pluto-simulation.md

```diff
@@ -7,5 +7,8 @@ This example was used in the workshop [MRI: Processing your own data](https://gi
 The participants used **KomaMRI** within a **Pluto** notebook to simulate some basic 1D sequences:  Free Induction Decay (FID), Gradient Echo (GE), and Spin Echo (SE). Try to solve the problems yourself! 😼
 
 ```@raw html
-<iframe type="text/html" src="../../../examples/pluto/simulation-example-empty.html" style="height:100vh;width:100%;"></iframe>
-
+<iframe type="text/html" src="../../../examples/educational_pluto_notebook/simulation-example-empty.html" style="height:100vh;width:100%;"></iframe>
+<details><summary>View Solution</summary>
+<iframe type="text/html" src="../../../examples/educational_pluto_notebook/simulation-example.html" style="height:100vh;width:100%;"></iframe>
+</details>
```

Comment:
Folder name missing number `7.educational_pluto_notebook`

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/253/commits/bc6b02f796cffa9a79a465bef5007a56ca538591